portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+        backgroundColor: 'white', 
+    },
+    titleLink: {
+        color: theme.palette.primary.dark,
+        marginRight: theme.spacing(1),
+    },
+    titleWrapper: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        marginBottom: theme.spacing(3),
+    },
+    operationListingBox: {
+        display: 'flex',
+        overflowY: 'scroll',
+    },
+    paper: {
+        display: 'flex',
+        flexDirection: 'column',
+        flex: 1
+    },
+    button: {
+        width: '200px'
+    }
+}));
+
+interface PolicyProps {
+    isCreateNew: boolean;
+    policyID: string | null;
+    disabled: boolean;
+}
+
+/**
+ * Renders the Global Policy management page.
+ * @param {boolean} isCreateNew - This value is true if form is for create new and false for edit.
+ * @param {string} policyID - This value is to indentify the policy (Null if creating a new one). 
+ * @param {boolean} disabled - This value is to disable the form (True if viewing the policy).
+ * @returns {TSX} - Policy management page to render.
+ */
+const Policies: FC<PolicyProps> =  ({
+    isCreateNew, 
+    policyID,
+    disabled,
+}) => {
+    const classes = useStyles();
+    const history = useHistory();
+    const [loading, setLoading] = useState(false);
+    const [policies, setPolicies] = useState<Policy[] | null>(null);
+    const [allPolicies, setAllPolicies] = useState<PolicySpec[] | null>(null);
+    const [name, setName] = useState('');
+    const [description, setDescription] = useState('');
+    const [appliedGatewayLabels, setAppliedGatewayLabels] = useState<string[]>([]);
+    const intl = useIntl();
+
+    /**
+     * Global level policy mapping. It will be initially empty.
+     */
+    const initGlobalLevelPolicy: GlobalLevelPolicy = {
+        request: [],
+        response: [],
+        fault: [],
+    }
+    const getInitGlobalLevelPoliciesState = () => {
+        return initGlobalLevelPolicy;
+    };
+    const [globalLevelPolicies, 
+        setGlobalLevelPolicies] = useState<GlobalLevelPolicy>(getInitGlobalLevelPoliciesState());
+
+    /**
+     * Fetches all common policies to front-end to show in the Policy List.
+     */
+    const fetchPolicies = () => {
+        const commonPoliciesPromise = API.getCommonOperationPolicies();
+        Promise.all([commonPoliciesPromise]).then((response) => {
+            const [commonPoliciesResponse] = response;
+            const commonPolicies = commonPoliciesResponse.body.list;
+            /**
+             * Similar to policies in Global Policies scenario.
+             * But as we are reusing PoliciesExpansion, both PolicySpec[] and Policy[] types are required.
+             */
+            setAllPolicies(commonPolicies);
+
+            commonPolicies.sort(
+                (a: Policy, b: Policy) => a.name.localeCompare(b.name))
+            
+            setPolicies(commonPolicies);
+        }).catch((/* error */) => {
+            // console.error(error); 
```

Comment:
```suggestion
            
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)